src/js/intlTelInput.js.ejs

```diff
@@ -246,6 +250,11 @@ Plugin.prototype = {
       // this is useful in lots of places
       this.countryListItems = this.countryList.children(".country");
     }
+    
+    // add the RTL class if requested
+    if(this.options.rtl === true) {
+    	this.telInputRootElement.addClass("iti-rtl");
```

Comment:
Ok, we also use it in the `destroy` method. If we're going to do this, we should do it properly. It should be set in `generateMarkup` when the root element is created, and let's call it `this.rootElement`. Please also update the `destroy` method to use it too. Thanks.